cypress/e2e/shared/checks.test.ts

```diff
@@ -115,6 +116,129 @@ describe('Checks', () => {
         }
       )
     })
+
+    it('can reconfigure a threshold check', () => {
+      const every = '5m'
+      const offset = '1m'
+      const testTags: {key: string; value: string}[] = [
+        {key: 'Prima', value: 'Smith'},
+        {key: 'Astaire', value: 'Rodgers'},
+        {key: 'Gable', value: 'Lombard'},
+      ]
+      const newMsg =
+        'Just a message from ${r._check_name} whoa level is ${r._level} ' +
+        'for type ${r._type} and ${r._source_measurement} ' +
+        'where foo is ${r.foo}'
+
+      // TODO use negative value after #3399 is resolved
+      // https://github.com/influxdata/ui/issues/3399
+      // const rangeBtm = "-0.01"
+      const rangeBtm = '0.01'
+      const rangeTop = '7.28'
+
+      initCheck(thresholdCheck).then(resp => {
+        expect(resp.name).to.equal(thresholdCheck.name)
+      })
+      cy.reload()
+      cy.intercept('PUT', '**/checks/*').as('putCheck')
+      cy.getByTestID('check-card--name')
+        .eq(0)
+        .click()
+      cy.getByTestID('overlay').should('be.visible')
+      // Properties
+      // Scheduler properties
+      cy.getByTestID('schedule-check').click()
+      cy.getByTestID('dropdown-menu--contents').within(() => {
+        cy.contains('5m').click()
+      })
+      cy.getByTestID('offset-options').click()
+      cy.getByTestID('dropdown-menu--contents').within(() => {
+        cy.contains('1m').click()
+      })
+      // Tags
+
+      for (let i = 0; i < testTags.length; i++) {
+        cy.getByTestID('dashed-button').click()
+        cy.getByTestID('tag-rule-key--input')
+          .eq(i)
+          .type(testTags[i].key)
+        cy.getByTestID('tag-rule-value--input')
+          .eq(i)
+          .type(testTags[i].value)
+      }
+
+      // Status Message Template
+
+      cy.getByTestID('status-message-textarea')
+        .clear()
+        .type(newMsg, {parseSpecialCharSequences: false})
+
+      // Thresholds
+      // remove one threshold
+      cy.getByTestID('panel-OK').within(() => {
+        cy.getByTestID('add-threshold-condition-OK').should('not.exist')
+        cy.getByTestID('dismiss-button').click()
+        cy.wait(500)
```

Comment:
`wait` is usually looked down upon. So, either move the dependent logic in here or just try moving this block's contents out and see if that works?

Reply:
possibly, try using `cy.focused()` instead if the upper suggestion doesn't work?

Reply:
The wait was an artefact from some debugging that I forgot to remove.  It is now removed.